build/cli/src/arg/wasm.rs

```diff
@@ -84,6 +106,14 @@ pub struct BuildInput {
     #[clap(long, arg_enum, enso_env())]
     pub profiling_level: Option<ProfilingLevel>,
 
+    /// Compiles Enso with given log level. If not set, defaults to [`warn`].
+    #[clap(long, arg_enum, enso_env())]
+    pub log_level: Option<LogLevel>,
```

Comment:
Rather than write what the default is in the documentation, I'd just remove `Option`, set `default_value_t` and let `clap` take care of presenting this,